hw/ip/rv_core_ibex/doc/_index.md

```diff
@@ -50,7 +50,22 @@ There are separate translations controls for instruction and data.
 Each control contains two programmable regions (2 for instruction and 2 for data).
 If a transaction matches multiple regions, the lowest indexed region has priority.
 
-For details on how to program the related registers, please see {{< regref "IBUS_ADDR_MATCHING0" >}} and {{< regref "IBUS_REMAP_ADDR0" >}}
+For details on how to program the related registers, please see {{< regref "IBUS_ADDR_MATCHING_0" >}} and {{< regref "IBUS_REMAP_ADDR_0" >}}
+
+## Random Number Generation
+
+The wrapper has a connection to the [Entropy Distribution Network (EDN)]({{< relref "hw/ip/edn/doc" >}}) with a register based interface.
+The {{< regref "RND_DATA" >}} register provides 32-bits directly from the EDN.
+The {{< regref "RND_STATUS" >}} register indicates if the data in {{< regref "RND_DATA" >}} is valid or not.
```

Comment:
Strictly speaking it's the RND_DATA_VALID bit of the RND_STATUS register, but I'm fairly sure we use this kind of shorthand in a lot of places, right?  It would be a bit more important to be specific if we add more bits to the RND_STATUS register.